### ISSUE TITLE
Script failed to scan birthdays. Error message: eyJ0eXBlIjoiU0NBTl9FUlJPUl9USU1FT1VUIn0=

### ISSUE THREAD

**Issue by Mike639** [zagushka/birthday-calendar-extractor#63]
Script failed to scan birthdays.
Error message: eyJ0eXBlIjoiU0NBTl9FUlJPUl9USU1FT1VUIn0=

**Comment by jier** [comment]
Same issue:
Script failed to scan birthdays.
Error message: eyJ0eXBlIjoiU0NBTl9FUlJPUl9USU1FT1VUIn0=

**Comment by mtsbh** [comment]
same here. On both chrome and vivaldi.

**Comment by amreggie** [comment]
I first got a different error code"
(Script failed to scan birthdays.
Error message: eyJlcnJvciI6IlJlbW92ZWQgaW5maW5pdGUgc2NhbiBsb2NrIiwidHlwZSI6IlNDQU5fRVJST1JfR0VORVJBTCJ9)

 Upon reading a post by GRANPA..something, I disabled Ad Blocker and changed language to English UK (from US). It now gives me THIS  error code 
Script failed to scan birthdays.
Error message: eyJ0eXBlIjoiU0NBTl9FUlJPUl9USU1FT1VUIn0=
